test/acceptance-tests/workspace-devtime/step-definitions/reading-artifacts/listModels.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+var util = require('util');
+var async = require('async');
+var app = require('../../../../../');
+var loopback = require('loopback');
+var path = require('path');
+var ModelConfig = app.models.ModelConfig;
+var exampleWorkspace = path.resolve(__dirname, '../../../../../example');
+
+module.exports = function() {
+  var numberOfExpectedModels = 0;
+  var numberOfAvailableModels = 0;
+  var testcase = this;
+
+  this.Given(/^I have a workspace containing (\d+) models$/, function(numberOfExpectedModels, next) {
+    testcase.numberOfExpectedModels = parseInt(numberOfExpectedModels);
+    next();
+  });
+
+  this.When(/^I list models for the workspace$/, function(next) {
+    ModelConfig.find(exampleWorkspace, function(err, list) {
+      testcase.numberOfAvailableModels = list.length;
+      next();
+    });
+  });
+
+  this.Then(/^All the model configs are returned$/, function(next) {
+    if (testcase.numberOfExpectedModels !== testcase.numberOfAvailableModels)
+      throw (new Error("This test didn't pass, numberOfAvailableModels is " + testcase.numberOfAvailableModels));
```

Comment:
should we use an assertion lib here? `var assert = require('assert')` at least? or maybe chai.expect:

```
expect(modelCount).to.equal(availableModels);
```